AI_Graphics/AI_Graphics/AIGraphics.cs

```diff
@@ -116,10 +116,16 @@ private IEnumerator Start()
             
             _inspector = new Inspector.Inspector(this);
 
-            // It takes some time to fully loaded in studio to save/load stuffs.
-            yield return new WaitUntil(() => {
-                return (KoikatuAPI.GetCurrentGameMode() == GameMode.Studio) ? KKAPI.Studio.StudioAPI.InsideStudio && _skyboxManager != null : true;
-            });
+            if (KoikatuAPI.GetCurrentGameMode() == GameMode.Maker) {
+                yield return new WaitUntil(() => {
+                    return (KoikatuAPI.GetCurrentGameMode() == GameMode.Maker && _skyboxGO == null);
+                });
+                StartCoroutine(Start()); // Start Loop for Maker
```

Comment:
isn't this a recursive call? we are in Start() itself, no?

Reply:
The reason why i inserted recursive call is Start() Coroutine is only called ONCE when the object has been initialized.
That means IBL will only can be used in first encounter of Character Maker.
As stated Issue #12, this makes IBL is not usable after closing character maker once.

So i did little trick, since this is async job, it will await till when IBL object is not valid in character maker.
After Awaiting Start() Coroutine detects Character Maker without IBL Instance, It will try to start Start() coroutine and await til it finds Character Maker without IBL Instance.